Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    display(rg_df)
</pre><p>Might be more readable if you are only expecting a single result to do this (get the first row, flip it and convert to a DF)</p><p>display(pd.DataFrame(rg_df.iloc[0].T))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>done!</p>